Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1267,6 +1267,22 @@ As per `k8s Service <https://kubernetes.io/docs/concepts/services-networking/ser
 Cilium's eBPF kube-proxy replacement by default disallows access to a ClusterIP service from outside the cluster.
 This can be allowed by setting ``bpf.lbExternalClusterIP=true``.
 
+VLAN 802.1q support
```

Comment:
The vlan support is not limited to the kube-proxy replacement, e.g., the host firewall feature benefits from it too (/cc @pchaigno to correct me if I'm wrong). Considering that, I think it makes sense to move this section to a documentation page dedicated to the datapath configuration (under the https://docs.cilium.io/en/stable/configuration/).

Reply:
Yes, you're right I think about benefits for host firewall. But I can't see any age related to datapath configuration - just only rate limiting.